src/modules/common/components/input-dropdown/input-dropdown.jsx

```diff
@@ -125,12 +125,11 @@ class InputDropdown extends Component {
             </option>
           ))}
         </select>
-        <i
-          className={classNames(Styles.InputDropdown__icon, "fa", {
-            "fa-angle-down": !this.state.showList,
-            "fa-angle-up": this.state.showList
-          })}
-        />
+        <span className={Styles.InputDropdown__icon}>
+          {!this.state.showList
+            ? ChevronDown({ stroke: "#FFFFFF" })
```

Comment:
can you use the ChevronFlip component for this ? Then you won't have to reuse code.

Reply:
updated two places to use ChevronFlip.